x.payment_batching/payment_batching.md

```diff
@@ -0,0 +1,307 @@
+# Payment Batching
+
+It's often possible to add more outputs to a transaction without
+increasing how many inputs it contains, meaning a single transaction can
+increase the number of people it pays faster than it proportionally
+grows in size.  That's scalability, and this chapter describes how
+high-frequency spenders can use this scaling technique of *payment
+batching* to reduce transaction sizes and fees by about 75% in
+practical situations.
+
+As of February 2019, payment batching is used by multiple popular
+Bitcoin services (mainly exchanges), is available as a built-in feature
+of many wallets (including Bitcoin Core), and should be easy to
+implement in custom wallets and payment-sending solutions.  On the
```

Comment:
I made a few changes based on this feedback, but not as much as you asked.  I think these are clearly downsides, but elsewhere in the document I've labeled them in the most prominent position as "concerns" rather than "problems" (while still using "problem" in more casual use).  I'm happy to discuss further.